src/dropdown.option.styles.ts

```diff
@@ -197,5 +200,15 @@ export default [
       display: block;
       inline-size: 0.875rem;
     }
+
+    .count-container {
+      font-size: var(--glide-core-typography-size-body-small);
+      padding-inline-end: var(--glide-core-spacing-base-sm);
```

Comment:
@ynotdraw is going to talk to Design about them abbreviating some variable names but not others.